Library/Homebrew/download_strategy.rb

```diff
@@ -388,6 +388,25 @@ def _fetch
   end
 end
 
+# Will convert "https://downloads.sourceforge.net/project/..." URLs to mirrorservice.org
+# In case environment variable HOMEBREW_NO_INSECURE_REDIRECT is set, so the Sourceforge is not ultimately broken
+class CurlSourceForgeSecureMirrorDownloadStrategy < CurlDownloadStrategy
+  def fetch
+    unless ENV["HOMEBREW_NO_INSECURE_REDIRECT"].nil?
+      ohai "Sourceforge HTTPS: #{@url}"
+      @url = https_mirror
+      ohai "=> converted to #{@url}"
```

Comment:
Just match this output format: https://github.com/smarek/homebrew/blob/sourceforge-download-strategy/Library/Homebrew/download_strategy.rb#L271-L276